deployments/cloudfoundry/buildpack/bin/supply

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -euo pipefail
+
+BUILD_DIR=$1
+CACHE_DIR=$2
+DEPS_DIR=$3
+IDX=$4
+
+BUILDPACK_DIR="$(readlink -f ${BASH_SOURCE%/*})"
+TARGET_DIR="$BUILD_DIR/.otelcollector"
+
+OTEL_CONFIG="${OTEL_CONFIG:-otelconfig.yaml}"
+# Version must be >=0.47.0, the first release included the enabled Cloud Foundry receiver
+OTEL_VERSION="${OTEL_VERSION:-0.47.0}"
+# Cloud Foundry only supports Linux stacks currently, but if that changes this variable can be modified
+# for the proper stack.
+OS="${OS:-linux_amd64}"
+
+echo "-----> Installing Otel Collector ${OTEL_VERSION}"
+echo "       BUILD_DIR: $BUILD_DIR"
+echo "       CACHE_DIR: $CACHE_DIR"
+echo "       DEPS_DIR: $DEPS_DIR"
+echo "       BUILDPACK_INDEX: $IDX"
+echo "       BUILDPACK_DIR: $BUILDPACK_DIR"
+echo "       TARGET_DIR: $TARGET_DIR"
+
+mkdir -p $TARGET_DIR
+
+# Copy over default OpenTelemetry configuration file for sidecar configuration use
+cp "$BUILDPACK_DIR/../$OTEL_CONFIG" "$TARGET_DIR/"
+
+# File name for given release version
+OTELCOL="${OTELCOL:-otelcontribcol_${OS}-v${OTEL_VERSION}}"
+OTELCOL_TAR="$OTELCOL.tar.gz"
+CACHED_OTELCOL_TAR="$CACHE_DIR/$OTELCOL_TAR"
+
+if [[ -f "$CACHED_OTELCOL_TAR" ]]; then
+  echo "-----> Using cached Otel Collector install: $CACHED_OTELCOL_TAR"
+  cp $CACHED_OTELCOL_TAR $TARGET_DIR
+else
+  OTELCOL_DOWNLOAD_URL="${OTELCOL_DOWNLOAD_URL:-https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v${OTEL_VERSION}/otelcol-contrib_${OTEL_VERSION}_${OS}.tar.gz}"
```

Comment:
Are there any plans to remove the executable from releases? I'm thinking of moving over to directly downloading the executable, instead of the current process of the `executable.tar.gz` (especially since there is no `executable.tar.gz` in our collector distribution). I just want to make sure my method is supported, and will continue to be since the upstream collector no longer directly includes executables.

Reply:
Alright, I'll work on moving over to use the Splunk distro release, and the executable binary itself rather than the upstream version.

Reply:
Latest commit implements using the Splunk distribution